views/room/roomTemplate.pug

```diff
@@ -0,0 +1,18 @@
+mixin roomTemplate(room, favorite, group)
+  .column.notification(class=`${!group ? 'is-success' : (!group.doNotDisturb? 'is-warning' : 'is-danger')}`)
+    .columns.is-vcentered.is-gapless
+      .column.is-narrow
+        span.icon.is-medium.has-text-warning.mr-2(onClick=`${favorite? `deleteFavorite(${room})` : `addFavorite(${room})`}`)
```

Comment:
Maybe create a `.is-clickable` class in `additions.scss`, and put it onto this span as well?

Recommendation:
```css
.is-clickable {
  cursor: pointer;
}
```